docs/CHAT_COMMANDS.md

```
@@ -0,0 +1,257 @@
+Slash chat-command system for the live chat. Lets viewers run built-in chat-command shortcuts (e.g. `/me`, `/roll`) and lets users define their own custom chat commands at two scopes:
+
+- **User-scoped** — created by a user, available to that user in any channel they chat in.
+- **Channel-scoped** — created by a streamer, available to any viewer chatting in that streamer's channel.
+
+Built-in chat commands are hardcoded on the client and always available.
+
+---
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)